test/e2e/framework/pod/resource.go

```diff
@@ -105,6 +105,7 @@ func (r ProxyResponseChecker) CheckAllResponses() (done bool, err error) {
 			Resource("pods").
 			SubResource("proxy").
 			Name(string(pod.Name)).
+			Suffix("/").
```

Comment:
this is unexpected

Reply:
see also https://github.com/kubernetes/kubernetes/pull/93144#issuecomment-659835179